crates/amaru-ledger/Cargo.toml

```diff
@@ -22,6 +22,7 @@ thiserror.workspace = true
 iter-borrow.workspace = true
 amaru-kernel.workspace = true
 amaru-ouroboros-traits.workspace = true
+sha3 = "0.10.8"
```

Comment:
nit: this version should probably be managed at the workspace level like other deps